script/e2e-test.sh

```diff
@@ -266,6 +266,10 @@ if [ "$USE_MULTICLUSTER_OIDC_ENV" = true ]; then
     "--set" "ingress.hostname=localhost"
     "--set" "ingress.tls=true"
     "--set" "ingress.selfSigned=true"
+    "--set" "ingress.annotations.\"kubernetes\.io\/ingress\.class\"=nginx"
+    "--set" "ingress.annotations.\"nginx\.ingress\.kubernetes\.io\/proxy-buffer-size\"=8k"
+    "--set" "ingress.annotations.\"nginx\.ingress\.kubernetes\.io\/proxy-read-timeout\"=600.0"
+    "--set" "ingress.annotations.\"nginx\.ingress\.kubernetes\.io\/proxy-buffers\"=4.0"
```

Comment:
As Helm allows to pass multiple input values files, do you think that this would work? Tested locally and it did, who knows in the E2E CI?

```suggestion
    "--values" "${INGRESS_VALUES_FILE}"
```

Where `INGRESS_VALUES_FILE` would be the path to this Yaml file:

```
ingress:
  hostname: localhost
  tls: true
  selfSigned: true
  annotations:
    kubernetes.io/ingress.class: nginx
    nginx.ingress.kubernetes.io/proxy-buffer-size: 8k
    nginx.ingress.kubernetes.io/proxy-buffers: "4.0"
    nginx.ingress.kubernetes.io/proxy-read-timeout: "600.0"

```

In this way we don't have to escape everything.

Reply:
Oh, that's another approach, good idea. If you already have the changes locally, please push them to this branch (as maintainers can edit PRs) to see if it passes and we can also close this one. Really looking forward to clean up the bunch of open PRs we have, haha